### PR TITLE
KakaoBrain Custom Resnet9 || CIFAR10 traninig || Kakao Brain

### PR DESCRIPTION
**Pull request by wbaek**
We updated the training record to about 38 seconds by using multi gpus(4 Tesla V100)
And updated to 68 seconds by single gpu (1 Tesla V100).

### CODE REVIEW

File: CIFAR10/train/kakaobrain_custom-resnet9_1v100-braincloud_pytorch.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "clint@KakaoBrain",
+    "authorEmail": "clint.b@kakaobrain.com",
+    "framework": "PyTorch 1.1.0",
+    "codeURL": "https://github.com/wbaek/torchskeleton/releases/tag/v0.1.1_dawnbench_cifar10_release",
+    "model": "Kakao Brain Custom ResNet9",
+    "hardware": "Tesla V100 * 1 GPU / 488 GB / 56 CPU (Kakao Brain BrainCloud)",
+    "timestamp": "2019-08-05",
+    "misc": {}
```
Comment (codyaustun) [inline CIFAR10/train/kakaobrain_custom-resnet9_1v100-braincloud_pytorch.json]: Can you add details on the learning rate schedule here?

File: CIFAR10/train/kakaobrain_custom-resnet9_4v100-braincloud_pytorch.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "clint@KakaoBrain",
+    "authorEmail": "clint.b@kakaobrain.com",
+    "framework": "PyTorch 1.1.0",
+    "codeURL": "https://github.com/wbaek/torchskeleton/releases/tag/v0.1.1_dawnbench_cifar10_release",
+    "model": "Kakao Brain Custom ResNet9",
+    "hardware": "Tesla V100 * 4 GPU / 488 GB / 56 CPU (Kakao Brain BrainCloud)",
+    "timestamp": "2019-08-05",
+    "misc": {}
```
Comment (codyaustun) [inline CIFAR10/train/kakaobrain_custom-resnet9_4v100-braincloud_pytorch.json]: Can you add details on the learning rate schedule here?

**Review by codyaustun — COMMENTED** [review]
Looks good to me, but it would be good to summarize the learning rate schedule and hyperparameters under the `misc` field.

**Comment by wbaek** [comment]
@codyaustun 
We have added details about the learning rate schedule and environments.

**Comment by codyaustun** [comment]
Looks good to me. Thanks!
